tempodb/search/util.go

```diff
@@ -24,6 +25,26 @@ func GetVirtualTagValues(tagName string) []string {
 	return nil
 }
 
+func GetVirtualTagValuesV2(tagName string) []tempopb.TagValue {
+
+	switch tagName {
+	case traceql.IntrinsicStatus.String():
+		return []tempopb.TagValue{
+			{Type: "keyword", Value: traceql.StatusOk.String()},
```

Comment:
I think these are the only keywords returned so far. Need to settle on the type string here and above.

Reply:
is `status.code` an old way to refer to `status`?

Reply:
I think this was from a previous line of thinking.  Removed.